credoai/integration.py

```diff
@@ -76,14 +76,14 @@ def __init__(self,
                  metric_type,
                  value,
                  subtype="base",
-                 dataset_id=None,
+                 dataset_name=None,
```

Comment:
@aproxacs - changing to `dataset_name` instead of `id` as part of move away from ids.